README.md

```diff
@@ -70,6 +75,9 @@ module "org-config" {
   # Name of the S3 bucket in the master account that contains the VPC flow logs
   vpc_flowlogs_bucket_name = ""
 
+  # Specify whether the cloudtrail_s3_bucket_name/kinesis_stream_name/vpc_flowlogs_bucket_name are in master account or not
```

Comment:
Please move this up.. close to the bucket input.
Is possible, can you rename to some thing line `cloudtrail_s3_bucket_in_master`? so it matches with bucket name parameter? This allows the customer to infer the tight relationship.